tetris.py

```diff
@@ -297,35 +297,42 @@ def full_line_clear(self):
         """
         Clears the line once it's fully populated with blocks
         """
+        flash_list = []
```

Comment:
Maybe name this `full_lines`? It's not only for flashing, it's also used for deleting lines.